agent/functional_tests/tests/functionaltests_windows_test.go

```diff
@@ -354,3 +354,28 @@ func TestContainerHealthMetricsWithStartPeriod(t *testing.T) {
 	t.Skip("Not supported")
 	containerHealthWithStartPeriodTest(t, "container-health-windows")
 }
+
+func TestTwoTasksSharedLocalVolume(t *testing.T) {
+	agent := RunAgent(t, nil)
+	defer agent.Cleanup()
+	agent.RequireVersion(">=1.19.0")
+
+	// start writer task first
+	wTask, err := agent.StartTask(t, "task-shared-vol-write-windows")
+	require.NoError(t, err, "Register task definition failed")
+
+	// then reader task
+	rTask, err := agent.StartTask(t, "task-shared-vol-read-windows")
+	require.NoError(t, err, "Register task definition failed")
+
+	// clean up
+	wErr := wTask.WaitStopped(2 * time.Minute)
```

Comment:
Can you reuse the constants defined [here](https://github.com/aws/amazon-ecs-agent/blob/master/agent/functional_tests/tests/functionaltests_test.go)? Same below.

Reply:
Fixed.